starboard/nplb/time_zone_get_name_test.cc

```diff
@@ -41,6 +48,19 @@ TEST(SbTimeZoneGetNameTest, IsKindOfSane) {
   // ":Pacific/Kiritimati" is the western-most timezone at UTC+14.
 }
 
+TEST(SbTimeZoneGetNameTest, IsIANAFormat) {
+  const char* name = SbTimeZoneGetName();
+  // Almost all time zone ID is Continent(or Ocean)/City
+  EXPECT_THAT(name, MatchesRegex(".+/.+"));
+  // The time zone ID contains a Continent or Ocean name.
+  EXPECT_THAT(name,
```

Comment:
Can we just split the string with a "/" delimiter and directly check the first item it it exists that it is one of the Continent/Ocean names? That would be a stronger check.